locales/en.json

```diff
@@ -376,7 +376,7 @@
       }
     },
     "limiting": "Limiting {0}",
-    "mute_desc": "You will no longer see any posts from this user. They can still follow you and see your posts. They will not know that they are muted.",
+    "mute_desc": "`You will no longer see any posts from this user. They can still follow you and see your posts. They will not know that they are muted.`",
```

Comment:
Why wrapping this string?

Reply:
My mistake, it's a typo.
Don't know how/why it happens but it's fixed.